tests/clar.c

```diff
@@ -432,9 +433,10 @@ clar_test_run(void)
 }
 
 void
-clar_test_shutdown()
+clar_test_shutdown(const double elapsed)
```

Comment:
To me, it looks like the elapsed time could also be part of the `_clar` structure, no? That'd also avoid piling up all that function arguments in `clar_print_shutdown`, as we can simply pass in `_clar` there.

Reply:
I'm generally not too big fan of that, seems just a global variable; but given that that's the main direction we're going into we can do that :+1: